packages/carbon-graphs/tests/unit/controls/Graph/GraphAxes-spec.js

```diff
@@ -2018,6 +2018,43 @@ describe('Graph - Axes', () => {
         fetchElementByClass(styles.labelPopupTooltip),
       ).not.toBeNull();
     });
+    describe('when y2Axis is true', () => {
+      beforeEach(() => {
+        graph.destroy();
+        const axisObj = utils.deepClone(axisDefault);
+        axisObj.y2 = {
+          show: true,
+          label: 'y2 axis',
+          lowerLimit: 11,
+          upperLimit: 25,
+        };
+        graph = new Graph({ ...getAxes(axisObj) });
+      });
+      it('should add correct transform point to y-axis label shape container', () => {
```

Comment:
nit: it should add the correct

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/205/commits/526ab4a52f731e54bcac96b439132b8927d6a4c4#diff-b94df4792973fae4e85b7bb9b7e1d60b3cffc8fbf3decea70798688e7656c5bbR2033).